ArduCopter/mode_acro_heli.cpp

```diff
@@ -49,16 +49,16 @@ void ModeAcro_Heli::run()
         attitude_control->reset_rate_controller_I_terms();
         break;
     case AP_Motors::SpoolState::GROUND_IDLE:
-        // Landed
-        if (motors->init_targets_on_arming()) {
+        // If aircraft is landed, set target heading to current and reset the integrator
+        // Otherwise motors could be at ground idle for practice autorotation
+        if ((motors->init_targets_on_arming() && !motors->use_new_integrator_mgmt()) || (copter.ap.land_complete && motors->use_new_integrator_mgmt())) {
```

Comment:
modify parameter name.  Maybe !use_leaky_integrator()

Reply:
yes, that sounds good to me